test/services/notify/notify-email.service.test.js

```diff
@@ -0,0 +1,146 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+const Sinon = require('sinon')
+
+const { describe, it, afterEach, beforeEach } = (exports.lab = Lab.script())
+const { expect } = Code
+
+const config = require('../../../config/notify.config.js')
+
+const { NotifyClient } = require('notifications-node-client')
+
+const NotifyEmailService = require('../../../app/services/notify/notify-email.service.js')
+
+describe.only('Notify - Email service', () => {
+  const stubNotify = true // Used to perform integration tests with notify
```

Comment:
Worth a discussion on having this in the test. 

I did initially create two test files (basically identical) one for the 'unit' test wit the stub. And one for an 'integration' test. 

With this toggle we can assert that out implementation with notify is working correctly. If / when we update the notify package,  i think it would be useful to run these manually locally.

Reply:
Add this in as config